### ISSUE TITLE
Maybe add a few configurable options to the .json? 

### ISSUE THREAD

**Issue by lavanoid** [roxas75/rxTools#115]
Think it would be possible to add configurable options in system.json, for setting the system version label?

I searched through the source code and noticed that in configuration.h, the labels are hard coded (line 23 & 24: #define rxmode_emu_label "RX3D" & #define rxmode_sys_label "Ver.").

I also noticed that cfw.c does something with the variable and uses it in "rxtools/data/0004013800000002.bin", I assume it's when the firm is patched on the first time rxTools is ran? Therefore it would be harder to implement because it would require re-patching the firmware, correct (reinitialise "Data Suite")?

Please keep in mind that I'm not an expert in C/C++ so please don't shoot me >.<

**Comment by AlbertoSONIC** [comment]
Nope it's definitely easy! The label patch that gets done during the first boot sets the label to "Shit". Then at every rxmode boot, Shit is replaced by Ver. or RX3D. You just have to replace it with *what you want* and here you go ;)

**Comment by lavanoid** [comment]
Okey doke. So this can be added as a configuration in the system.json, right? I was previously doing it manually by just editing the source code then compiling rxTools but it sorta gets frustrating when my GitHub spazzes out when making a pull request 0~0

If someone does manage to implement this feature, I'd totally wanna check out the changes as I was trying to make sense of the code, earlier on :)

> if(FileOpen(&firm, "rxtools/data/0004013800000002.bin", 0)){
>		FileWrite(&firm, mode ? &nat_emuwrite : &mmc_original, 8, 0xCCF2C);
>		FileWrite(&firm, mode ? &nat_emuread : &mmc_original, 8, 0xCCF6C);
>		FileWrite(&firm, mode ? rxmode_emu_label : rxmode_sys_label, 4, 0x7A5A0);
>		FileClose(&firm);
>	}

I opened 0004013800000002.bin in HxD then went to 7A5A0, so I saw "RX3D". I'm learning :D Sorta know how it writes files, yay :P
